lib/components/TextField.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <input
+        ref="textfield" 
+        :placeholder="hint"
+        :disabled="!editable"
+        :maxlength="maxLength"
+        :type="secure ? 'password' : keyboardType"
+        :value="value"
+        :spellcheck="autoCorrect"
+        @blur="onBlur($event)"
+        @focus="onFocus($event)"
+        @keyup.enter="onReturnPress($event)"
+        @change="onTextChange($event)"
+        @input="updateValue()"
+    />
+</template>
+
+<script>
+export default {
+    name: 'TextField',
+    props: [
```

Comment:
missing "returnKeyType" prop and implementation

Reply:
I could not find the implementation of customizing the return key type in web.